_neural-search-plugin/index.md

```diff
@@ -0,0 +1,193 @@
+---
+layout: default
+title: Neural Search plugin
+nav_order: 1
+has_children: false
+has_toc: false
+---
+
+The Neural Search plugin is an experimental feature. For updates on the progress of the Neural Search plugin, or if you want to leave feedback that could help improve the feature, join the discussion in the [Neural Search forum](https://forum.opensearch.org/t/feedback-neural-search-plugin-experimental-release/11501).    
+{: .warning}
+
+The OpenSearch Neural Search plugin enables the integration of Machine Learning (ML) language models into user's search workloads. During ingestion and search, the Neural Search plugin transforms text into vectors. Then, Neural Search uses the transformed vectors in vector-based search."
```

Comment:
Just noticed after merging that there is an extra quotation mark at the end of this sentence.